scripts/dl_cleanup.py

```diff
@@ -191,25 +194,20 @@ def main(argv):
     try:
         (opts, args) = getopt.getopt(
             argv[1:],
-            "hdBw:",
+            "hdBwD:",
             [
                 "help",
                 "dry-run",
                 "show-blacklist",
                 "whitelist=",
+                "download-dir=",
             ],
         )
-        if len(args) != 1:
-            usage()
-            return 1
     except getopt.GetoptError as e:
         usage()
         return 1
-    directory = args[0]
```

Comment:
To be compatible with the buildbot we should add this legacy behaviour. Meaning, if `-D` is not defined assume `arg[0]`. What do you think?

Reply:
buildbot use this script?

so the problem is that buildbot would still provide the args and would cause some problem? 
I think that's the only way to solve this. Correct?